### PR TITLE
Version-bump-v2.7.9

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
= 2.7.9 - WEDNESDAY, 13th SEPTEMBER 2023 =
* Improvement: InfoBox - Changed Info Box icon position dropdown to multi control.
* Fix: Call To Action - Resolved an issue wherein the buttons would not be aligned as intended.
* Fix: How To - Resolved an issue in which the responsive heading font-sizes would not work as intended.
* Fix: Image gallery - Resolved an issue wherein the lightbox was not working correctly within container.
* Fix: Post Blocks - Resolved an issue wherein text-formatting for buttons would appear as HTML tags on the front-end.
* Fix: Price List - Resolved issues wherein the title bottom margin, and the row and column gaps would not work as intended for responsive devices.

- [x]  Run to update version number : `grunt bump-version --ver=[version-number]`
- [x]  Verify `Stable tag` is [version-number] in readme.txt
- [x]  Verify `Tested upto` is set to latest tested version of WordPress
- [x]  Verify changelog `date` and `content` as per Spectra standards
- [x]  Generate README.md : `grunt readme`
- [x]  Update `version` in `ultimate-addons-for-gutenberg.php` in plugin description
- [x]  Verify constant `UAGB_VER` in `class-uagb-loader.php` with latest version of Spectra
- [x]  Generate POT file : `npm run makepot`
- [x]  Update version in `package.json `and `package-lock.json`

### CODE REVIEW

File: README.md
Comment (bsf-zanev) [inline README.md:1]: @rajmohanreddy774 - Could you update the changelog?
I've improved a few entries.

```
* Improvement: Info Box - Changed the Info Box icon position dropdown to a multi-control.
* Fix: Call To Action - Resolved an issue wherein the buttons would not be aligned as intended.
* Fix: How To - Resolved an issue in which the responsive heading font-sizes would not work as intended.
* Fix: Image Gallery - Resolved an issue wherein the lightbox would not work intended within Containers.
* Fix: Post Blocks - Resolved an issue wherein text-formatting for buttons would appear as HTML tags on the front-end.
* Fix: Price List - Resolved an issue in which the title bottom margin, row, and column gaps would not work as intended in the responsive editor.
```

Comment (adi3890) [reply README.md:1]: updated @bsf-zanev can you please re-review

**Review by adi3890 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -215,16 +215,16 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
...
 ## Changelog ##
 
-### x.x.x ###
-* Improvement: InfoBox - Changed Info Box icon position dropdown to multi control.
+### 2.7.9 - WEDNESDAY, 13th SEPTEMBER 2023 ###
+* Improvement: Info Box - Changed the Info Box icon position dropdown to a multi-control.
 * Fix: Call To Action - Resolved an issue wherein the buttons would not be aligned as intended.
 * Fix: How To - Resolved an issue in which the responsive heading font-sizes would not work as intended.
-* Fix: Image gallery - Resolved an issue wherein the lightbox was not working correctly within container.
+* Fix: Image Gallery - Resolved an issue wherein the lightbox would not work intended within Containers.
```
[the remark this answers is not in the document]
Comment (bsf-zanev) [reply README.md:222]: CC - @rajmohanreddy774

File: readme.txt [hunk 2]
```diff
@@ -215,16 +215,16 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
...
 == Changelog ==
 
-= x.x.x =
-* Improvement: InfoBox - Changed Info Box icon position dropdown to multi control.
+= 2.7.9 - WEDNESDAY, 13th SEPTEMBER 2023 =
+* Improvement: Info Box - Changed the Info Box icon position dropdown to a multi-control.
 * Fix: Call To Action - Resolved an issue wherein the buttons would not be aligned as intended.
 * Fix: How To - Resolved an issue in which the responsive heading font-sizes would not work as intended.
-* Fix: Image gallery - Resolved an issue wherein the lightbox was not working correctly within container.
+* Fix: Image Gallery - Resolved an issue wherein the lightbox would not work intended within Containers.
```
Comment (bsf-zanev) [inline readme.txt:222]:
```suggestion
* Fix: Image Gallery - Resolved an issue wherein the lightbox would not work as intended within Containers.
```

**Review by bsf-zanev — APPROVED** [review]

**Pull request merged**